OpenRA.Game/FileSystem/Folder.cs

```diff
@@ -18,39 +18,37 @@ namespace OpenRA.FileSystem
 {
 	public sealed class Folder : IReadWritePackage
 	{
-		readonly string path;
+		public string Name { get; }
```

Comment:
Bit of a scope creep, but why did we expose that as `Name` and not `Path`? (And now you won't even be able to tell be looking at the previous "getter".)

Reply:
It's `Name` to implement `IReadOnlyPackage.Name`. Do we want to rename the interface and all implementations to `Path`? That feels like a larger scope change than you may have intended with your suggestion.

Reply:
Yeah it is probably best left for another PR if we want to change this. `Name` just implies to me that this is the package name, not the entire path.